Cargo.lock

```diff
@@ -252,12 +252,13 @@ dependencies = [
 
 [[package]]
 name = "alloy-json-rpc"
-version = "1.0.9"
+version = "1.0.23"
```

Comment:
Are those required or can we remove the updates?

Reply:
These seem to be required, I suspect the change was precipitated by a dependency on `alloy-transport-ipc = "1.0.23"`